elk/training/sweep.py

```diff
@@ -127,52 +110,42 @@ def execute(self):
                 # single sweep.
                 train_datasets = tuple(ds.strip() for ds in dataset_str.split("+"))
 
-                for var_weight in weights:
-                    for neg_cov_weight in weights:
-                        out_dir = sweep_dir / model / dataset_str
+                out_dir = sweep_dir / model / dataset_str
 
-                        data = replace(
-                            self.run_template.data, model=model, datasets=train_datasets
-                        )
-                        run = replace(self.run_template, data=data, out_dir=out_dir)
-                        if var_weight is not None and neg_cov_weight is not None:
-                            assert isinstance(run.net, EigenFitterConfig)
-                            run.net.var_weight = var_weight
-                            run.net.neg_cov_weight = neg_cov_weight
-
-                            # Add hyperparameter values to output directory if needed
-                            assert run.out_dir is not None
-                            run.out_dir /= f"var_weight={var_weight:.2f}"
-                            run.out_dir /= f"neg_cov_weight={neg_cov_weight:.2f}"
-
-                        try:
-                            run.execute()
-                        except torch.linalg.LinAlgError as e:
-                            print(colorize(f"LinAlgError: {e}", "red"))
+                data = replace(
+                    self.run_template.data, model=model, datasets=train_datasets
+                )
+                run = replace(self.run_template, data=data, out_dir=out_dir)
+
+                # Add hyperparameter values to output directory if needed
+                assert run.out_dir is not None
+
+                try:
+                    run.execute()
+                except torch.linalg.LinAlgError as e:
+                    print(colorize(f"LinAlgError: {e}", "red"))
```

Comment:
Wait we can remove this now bc we're not doing eigendecomposition right?

Reply:
Fortunately yes.